packages/app-headless-cms/src/admin/views/contentEntries/ContentEntry/FullScreenContentEntry/FullScreenContentEntryHeaderLeft.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useMemo } from "react";
+import { ReactComponent as BackIcon } from "@material-design-icons/svg/round/arrow_back.svg";
+import { useNavigateFolder } from "@webiny/app-aco";
+import { IconButton } from "@webiny/ui/Button";
+import { Typography } from "@webiny/ui/Typography";
+import { useModel } from "~/admin/components/ModelProvider";
+import { useContentEntry } from "~/admin/views/contentEntries/hooks";
+import {
+    EntryMeta,
+    EntryTitle,
+    EntryVersion,
+    TitleWrapper,
+    EntryName
+} from "./FullScreenContentEntry.styled";
+
+export const FullScreenContentEntryHeaderLeft = () => {
+    const { model } = useModel();
```

Comment:
This hook is redundant, because `useContentEntry()` hook below, has access to that same model, just add a `contentModel` key.